### ISSUE TITLE
Does this work anymore ? 

### ISSUE THREAD

**Issue by Adalind-no-commit-git** [Shimoro-Rune/NitroPerks#35]
the effort put into this is nice. But it seems like it doesnt work anymore.

**Comment by FadZZer** [comment]
no

**Comment by Bettlee** [comment]
> the effort put into this is nice. But it seems like it doesnt work anymore.

Yeah, the creator hasn't worked on it since. I decided to remake the plugin
